.github/workflows/release.yml

```
@@ -0,0 +1,91 @@
+            
+            ### 📋 Changes in this release:
+            ${{ steps.changelog.outputs.changelog }}
+            
+            ### 🔧 Technical Details:
+            - **Branch**: ${{ github.ref_name }}
+            - **Commit**: ${{ github.sha }}
+            - **Build**: ${{ github.run_number }}
+            
+            ### 📦 Deployment:
+            Download the deployment package from the [CI/CD workflow artifacts](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)